boltdesignsystem/bolt
# DS-609 Custom icon workflow

## Jira

https://pegadigitalit.atlassian.net/browse/DS-609

## Summary

The icon element was updated to allow add custom SVGs as twig files.

## Details

- The `icon_custom_path` prop was added to the icon element schema
- The Icon element twig template was updated
- Documentation was updated

## How to test

- Pull the branch to your local.
- Check if the icon is rendering when using Twig include.
- Review the docs.

## Release notes

A new `custom_icon_path` was added to make it possible to render the icon element by Twig usage.
